codec_specs.md

```diff
@@ -182,6 +182,26 @@ Description: The Matroska video stream will contain a demuxed Elementary Stream
 
 Initialization: none
 
+### V_MPEG4/ISO/AVC
+
+Codec ID: V_MPEG4/ISO/AVC
+
+Codec Name: AVC/H.264
+
+Description: Individual pictures of AVC/H.264 stored as described in ISO/IEC 14496-15.
+
+Initialization: The `Private Data` contains a `AVCDecoderConfigurationRecord` structure as defined in ISO/IEC 14496-15, optionally followed by 1 or more extension blocks; each extension block begins with a 4-byte extension block size field which is the size of the extension block minus 4 (excluding the size of the extension block size field), followed by a 4-byte extension block identifier field; extension block identifier field MAY be identical to the ones found in ISO/IEC 14496-12 and its extensions.
```

Comment:
I guess, the reviewers prefer `followed by one or more` rather than `followed by 1 or more`.